kt/plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/tasks/android/checkAndroidJarAccessibleTask.kt

```diff
@@ -0,0 +1,28 @@
+package godot.gradle.tasks.android
+
+import godot.gradle.projectExt.godotJvmExtension
+import org.gradle.api.Project
+import org.gradle.api.Task
+
+fun Project.checkAndroidJarAccessibleTask(): Task {
+    return with(tasks.create("checkAndroidJarAccessible")) {
```

Comment:
Not a big issue, but have you tried using `register` instead of `create`? With register a task is only configured/resolved when it will participate in the execution.

Reply:
Ah nice to know. I'll change them